src/components/frame/v5/pages/BalancePage/partials/BalanceTable/hooks.tsx

```diff
@@ -116,6 +123,9 @@ export const useBalanceTableColumns = (
         cell: ({ row }) => {
           if (!row.original.token) return [];
 
+          if (row.original.loading)
+            return <div className="h-4 w-40 skeleton" />;
```

Comment:
```suggestion
          if (row.original.loading) {
            return <div className="h-4 w-40 skeleton" />;
          }
```

But that's more of a nitpick than anything else

Reply:
And it applies to all other instances were you've used this pattern. 

Feel free to ignore as this is just a code style difference of oppinion

Reply:
Ah, I actually prefer with brackets too, I must've just done it quickly and it didn't register.

Also; I should change it as a matter of standards! https://github.com/JoinColony/ColonyFrontEndLivingStandard/blob/master/codeQuality/index.md#explicitly-scoping-if-statements